operator/templates/index.html

```diff
@@ -67,6 +64,54 @@ <h5>
                         </label>
                     </div>
                 </div>
+                <br />
+                <h5>
+                    Track
+                </h5>
+                <div>
+                    <div class="custom-control custom-radio">
+                        <input class="custom-control-input" type="radio" name="track-select" id="TrainingMap" value="TrainingMap">
+                        <label class="custom-control-label" for="TrainingMap">
+                            TrainingMap
```

Comment:
Yes, we are hard-coding all the maps because at this moment there is no way to retrieve these automagically.